### ISSUE TITLE
Error info incomplete

### ISSUE THREAD

**Issue by c10b10** [steelbrain/flow-ide#68]
In atom, the only displayed error message is the one line 50.  The only way to get to the useful information is to run flow in the terminal, and to see the full output, which gives me more clues about the issue, namely the "error" type issue. 

```sh
src/auth/auth.reducer.js:50
 50:       return {
                  ^ object literal. This type is incompatible with the expected return type of
 34: export default function auth (state: State = initializeState(), action: Action):State {
                                                                                     ^^^^^ object type
  Property `error` is incompatible:
     26:   error: null
                  ^^^^ null. This type is incompatible with
     11:   error: Object
                  ^^^^^^ object type
```
Is there a way to show full output? (For the 26 and 11 line to be highlighted in addtion / instead of the 50 one). Seems to me that the errors on 26 / 11 are more useful.

**Comment by steelbrain** [comment]
Hi @c10b10 

Flow is actually pretty stupid when it comes to providing useful traces but I'll try to do something about it, Thanks for the request

**Comment by c10b10** [comment]
Something similar to the "Tuples" example interaction from here: https://flowtype.org/docs/quick-reference.html#primitives  would be brilliant.

Notice that hovering over "string" from line 1 / "false" from line 15, highlights the the associated line token.

**Comment by lloiser** [comment]
@c10b10 This is most likely fixed with the upcoming pull request #94. Stay tuned!
